src/test/java/org/amazon/base/BaseMethods.java

```diff
@@ -0,0 +1,45 @@
+package org.amazon.base;
```

Comment:
Name packages wisely. Someone who works for amazon would probably write their packages with such naming convention like `org.amazon` - read the rules for creating good package names and make changes accordingly

Reply:
Done.